### ISSUE TITLE
Enrich Private Key Verification With Driftwood output

### ISSUE THREAD

**Issue by dxa4481** [trufflesecurity/trufflehog#307]
**Is your feature request related to a problem? Please describe.**
Today when Driftwood gets a match TruffleHog just says "verified". It would be great if it included the rest of the Driftwood output, what SSL cert, or Github user is affected, etc..

**Describe the solution you'd like**
See above

**Comment by nyanshak** [comment]
If you use `--json`, you will get formatted Driftwood output in the `StructuredData` field:

```bash
trufflehog filesystem ./leaky-test-repo --json | jq .
...
{
  ...
  "StructuredData": {
    "github_ssh_key": [
      {
        "user": "my-test-user-redacted"
      }
    ]
  }
}
```

This structured data is not written out in other output formats (e.g., text to stdout with no flags, or with `--json-legacy`).

It would be nice if there was an option to include this structured data in the normal trufflehog output as well.

**Comment by dustin-decker** [comment]
Closed by #1760
